docs/dev/new_check_howto.md

```diff
@@ -139,22 +147,21 @@ class AwesomeCheck(AgentCheck):
                 self.service_check('awesome.search', self.WARNING)
 ```
 
-To learn more about the base Python class, see [Anatomy of a Python Check][6].
+To learn more about the base Python class, see [Anatomy of a Python Check][9].
 
-### Writing tests
+## Write tests
 
 There are two basic types of tests:
-
 - Unit tests for specific functionality.
 - Integration tests that execute the `check` method and verify proper metrics collection.
 
-Tests are _required_ if you want your integration to be included in `integrations-extras`. **Note**: [pytest][7] and [tox][8] are used to run the tests.
+Tests are required if you want your integration to be included in `integrations-extras`.
 
-For more information, see the [ddev installation guide][9].
+**Note**: [pytest][10] and [tox][11] are used to run the tests.
```

Comment:
FYI, we are switching from `tox` to [`hatch`](https://github.com/pypa/hatch). This is a work in progress in `integrations-core` and we plan to migrate extras integration after.

Reply:
If the tool is creating integrations that use hatch, I think it make sense to update this now 👍